src/MarginTrading.SettingsService.Contracts/Scheduling/ScheduleConstraintContract.cs

```diff
@@ -2,6 +2,9 @@
 
 namespace MarginTrading.SettingsService.Contracts.Scheduling
 {
+    /// <summary>
+    /// Either Date or DayOfWeek must be set
```

Comment:
I am not sure about it at all.. How every nights will be configured is such case? As I understood general schedule is From Monday to Friday, from 10:00 to 19:00. We will create a rule for every working day?
By the way @alpo-8 attached ScheduleSettingsExampleDaily.json example to the task...